src/Microsoft.VisualStudio.Composition/Reflection/ResolverExtensions.cs

```diff
@@ -182,32 +178,9 @@ public static MemberInfo Resolve(this MemberRef memberRef)
             throw new NotSupportedException();
         }
 
+        [Obsolete("Use " + nameof(MemberRef) + " instead.", error: true)]
         public static MemberInfo Resolve(this MemberDesc memberDesc)
```

Comment:
This method was always just broken. And we never used it. So rather than fix it (which would require a breaking change), since it was never used I just scrape out its implementation. I'll remove these broken and unused types in v16.